index.ipynb

```diff
@@ -12,65 +12,65 @@
     "\n",
-    "[![hohmann_bielliptic](docs/source/examples/hohmann_bielliptic.png)](docs/source/examples/Comparing%20Hohmann%20and%20bielliptic%20transfers.ipynb)\n",
+    "[![hohmann_bielliptic](docs/source/examples/hohmann_bielliptic.png)](docs/source/examples/Comparing%20Hohmann%20and%20bielliptic%20transfers.md)\n",
     "\n",
-    "## [Exploring the New Horizons launch](docs/source/examples/Exploring%20the%20New%20Horizons%20launch.ipynb)\n",
+    "## [Exploring the New Horizons launch](docs/source/examples/Exploring%20the%20New%20Horizons%20launch.md)\n",
     "\n",
-    "[![new_horizons](docs/source/examples/new_horizons.png)](docs/source/examples/Exploring%20the%20New%20Horizons%20launch.ipynb)\n",
+    "[![new_horizons](docs/source/examples/new_horizons.png)](docs/source/examples/Exploring%20the%20New%20Horizons%20launch.md)\n",
     "\n",
-    "## [Going to Jupiter with Python using Jupyter and poliastro](docs/source/examples/Going%20to%20Jupiter%20with%20Python%20using%20Jupyter%20and%20poliastro.ipynb)\n",
+    "## [Going to Jupiter with Python using Jupyter and poliastro](docs/source/examples/Going%20to%20Jupiter%20with%20Python%20using%20Jupyter%20and%20poliastro.md)\n",
     "\n",
-    "[![mars](docs/source/examples/jupiter.png)](docs/source/examples/Going%20to%20Jupiter%20with%20Python%20using%20Jupyter%20and%20poliastro.ipynb)\n",
+    "[![mars](docs/source/examples/jupiter.png)](docs/source/examples/Going%20to%20Jupiter%20with%20Python%20using%20Jupyter%20and%20poliastro.md)\n",
     "\n",
-    "## [Going to Mars with Python using poliastro](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.ipynb)\n",
+    "## [Going to Mars with Python using poliastro](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.md)\n",
     "\n",
-    "[![mars](docs/source/examples/trans_30_260.png)](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.ipynb)\n",
+    "[![mars](docs/source/examples/trans_30_260.png)](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.md)\n",
     "\n",
-    "## [Propagation using Cowell's formulation](docs/source/examples/Propagation%20using%20Cowell's%20formulation.ipynb)\n",
+    "## [Propagation using Cowell's formulation](docs/source/examples/Propagation%20using%20Cowell's%20formulation.md)\n",
     "\n",
-    "[![cowell](docs/source/examples/cowell.png)](docs/source/examples/Propagation%20using%20Cowell's%20formulation.ipynb)\n",
+    "[![cowell](docs/source/examples/cowell.png)](docs/source/examples/Propagation%20using%20Cowell's%20formulation.md)\n",
     "\n",
-    "## [Revisiting Lambert's problem in Python](docs/source/examples/Revisiting%20Lambert's%20problem%20in%20Python.ipynb)\n",
+    "## [Revisiting Lambert's problem in Python](docs/source/examples/Revisiting%20Lambert's%20problem%20in%20Python.md)\n",
```

Comment:
This must be:

```markdown
docs/source/examples/Revisiting%20Lamberts%20problem%20in%20Python.md
```

Reply:
Sure thing, fixing this in the next commit